README.md

```diff
@@ -643,15 +268,18 @@ based library in JavaScript.
 ## Documentation
 Here is a list of all documentation related to drand: 
 
-* For a high level presentation of motivations and background, here are some public
+- To learn more about the protocol, the motivation and its backgronund
+  - For a high level presentation of motivations and background, here are some public
   [slides](https://docs.google.com/presentation/d/1t2ysit78w0lsySwVbQOyWcSDnYxdOBPzY7K2P9UE1Ac/edit?usp=sharing)
   about drand or online [video](https://www.youtube.com/watch?v=ydwW2HFFxNI&list=PLhuBigpl7lqu6xWpiXtbEzJQtlMH1tqoG&index=3).
-* The client-side API documentation of drand:
-  [link](https://hackmd.io/@nikkolasg/HJ9lg5ZTE) 
-* The drand *operator guide* documentation:
-  [link](https://hackmd.io/@nikkolasg/Hkz2XFWa4) 
-* A basic explainer of the cryptography behind drand:
-  [link](https://hackmd.io/@nikkolasg/HyUAgm234), 
+  - [A basic explainer of the cryptography behind drand](https://hackmd.io/@nikkolasg/HyUAgm234), 
```

Comment:
@nikkolasg any reason why to keep this in a hackmd? Mind if I pull it to this repo so that others can contribute to it as well?